index.html

```diff
@@ -185,11 +185,14 @@ <h3>Skills</h3>
                 </li>
             </ul>
         </section>
-        <section class="form-container">
+        <section id="contact" class="form-container">
             <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
             <form action="https://formspree.io/f/mjvdkwqz" method="post" class="contact-form">
+                <label for="user-name" hidden >Full name</label>
                 <input type="text" maxlength="30" placeholder="Full name" class="input-field" name="user-name" required>
+                <label for="user-email" hidden>Email</label>
                 <input type="email" placeholder="Email" class="input-field" name="user-email" required>
+                <label for="user-text" hidden>Message</label>
                 <textarea name="user-text" maxlength="500" placeholder="Enter Text here" class="textarea-field" required></textarea>
                 <button type="submit" class="submit-btn">Get in touch</button>
             </form>
```

Comment:
- [ ] Your button has not had enough contrast of 4.5:1. Please add enough contrast for this button :+1:

![Screenshot from 2023-01-25 00-12-51](https://user-images.githubusercontent.com/80458199/214433582-8dd52727-2e10-4321-bf41-b55ec3c84f2b.png)


![Screenshot from 2023-01-25 00-13-08 (1)](https://user-images.githubusercontent.com/80458199/214433578-a01b7734-a601-4b15-a751-91690bf626b0.png)